dask/dataframe/tests/test_reshape.py

```diff
@@ -109,21 +110,36 @@ def decorator():
     return decorator
 
 
+@contextlib.contextmanager
+def ignore_numpy_bool8_deprecation():
+    # This warning comes from inside `pandas`. We can't do anything about it, so we ignore the warning.
+    # Note it's been fixed upstream in `pandas` https://github.com/pandas-dev/pandas/pull/49886.
+    with warnings.catch_warnings():
+        warnings.filterwarnings(
+            "ignore",
+            category=DeprecationWarning,
+            message="`np.bool8` is a deprecated alias for `np.bool_`",
+        )
+        yield
```

Comment:
You might prefer https://github.com/dask/distributed/commit/3ac8631f6378124307c588019560aa8e269fcc3f instead

Reply:
I considered that, but since this warning was only showing up in three tests I decided to go for a more surgical approach. Just a personal preference, when it's reasonable from a maintenance perspective

Reply:
I disagree with your approach but am thrilled to defer 🙂